### PR TITLE
CI: Validate source formatting with prettier

### PR DESCRIPTION
**Pull request by gavacho**

### CODE REVIEW

**Comment by gavacho** [comment]
This will configure travis-ci to test that the source code has been correctly formatted with prettier.

To merge this, it looks like we'd have to:
1) Configure prettier to ignore files we don't author (like jquery) -- easiest way to do this would be to move them into a directory called `vendor` or something.
2) Use prettier to format the unformatted files

**Comment by lusbenjamin** [comment]
I like that standardized formatting makes it faster to understand shared code. I dislike losing time to [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality). I lean slightly to some kind of approach that uses prettier `--write` automatically because I've gotten used to relying on transpiling source code anyway.

**Comment by lusbenjamin** [comment]
I like @gavacho's more precise `npm` scripts, but note that this will conflict with #333

**Comment by gavacho** [comment]
I've rebased of master and addressed the conflicts

File: package.json [hunk 1]
```diff
@@ -51,10 +51,11 @@
   "scripts": {
     "start": "electron .",
     "postinstall": "install-app-deps",
-    "lint": "npx eslint . --quiet",
-    "pretest": "npm run lint",
-    "test": "npx jest",
-    "predist": "npm test",
+    "format": "prettier --write \"**/*.js\"",
+    "test": "npm run --silent test:eslint && npm run --silent test:prettier && npm run --siltent test:jest",
```
Comment (lusbenjamin) [inline package.json]: `npm run --siltent test:jest` ==> `npm run --silent test:jest`

Comment (gavacho) [reply package.json]: thanks for catching this!

**Comment by lusbenjamin** [comment]
🤣 can confirm that PR works as intended. build only failed because we merged immediately after first build before 2nd could succeed.

**Pull request merged**